packages/ic-response-verification-tests/wasm-tests/main.ts

```diff
@@ -49,6 +50,7 @@ async function main(): Promise<void> {
     await v2Test(principal, agent, actor);
   } catch (error) {
     console.error('Error running e2e tests...', error);
+    exit(1);
```

Comment:
If we encounter an error, make sure to exit with a non-zero exit code so that the CI script picks up on it.